clients/privacy-center/cypress/e2e/banner-overlay-dismissal.cy.ts

```diff
@@ -1,309 +1,173 @@
-import { ConsentMethod } from "fides-js";
+import { ConsentMethod, CONSENT_COOKIE_NAME, FidesCookie } from "fides-js";
 import { stubConfig } from "../support/stubs";
 
-function assertDismissCalled() {
-  cy.get("@FidesUpdated")
-    .should("have.been.calledOnce")
-    .its("lastCall.args.0.detail.extraDetails.consentMethod")
-    .then((consentMethod) => {
-      expect(consentMethod).to.eql(ConsentMethod.dismiss);
-    });
-
-  cy.wait("@patchPrivacyPreference");
-}
-
-describe("Banner and overlay dismissal", () => {
-  describe("Consent with dismissal allowed", () => {
-    beforeEach(() => {
-      stubConfig({
-        options: {
-          tcfEnabled: false,
-          preventDismissal: false,
-        },
+describe("Banner and modal dismissal", () => {
+  // Helper function for some test case assertions
+  function assertDismissCalled() {
+    cy.get("@FidesUpdated")
+      .should("have.been.calledOnce")
+      .its("lastCall.args.0.detail.extraDetails.consentMethod")
+      .then((consentMethod) => {
+        expect(consentMethod).to.eql(ConsentMethod.DISMISS);
+      });
+
+    cy.waitUntilCookieExists(CONSENT_COOKIE_NAME).then(() => {
+      cy.getCookie(CONSENT_COOKIE_NAME).then((cookie) => {
+        const fidesCookie: FidesCookie = JSON.parse(
+          decodeURIComponent(cookie!.value)
+        );
+        expect(fidesCookie.fides_meta)
+          .property("consentMethod")
+          .is.eql(ConsentMethod.DISMISS);
```

Comment:
Added this to the assertion to check that the cookie records the consent method too.